tests/fast-integration/telemetry-test/telemetry-test.js

```diff
@@ -80,7 +80,9 @@ describe('Telemetry Operator tests', function() {
       assert.fail('Should not be able to apply invalid LogPipeline');
     } catch (e) {
       assert.equal(e.statusCode, 403);
-      expect(e.body.message).to.have.string('denied the request', 'Invalid indentation level');
+      expect(e.body.message).to.have.string('denied the request');
+      const errMsg = 'section \'abc\' tried to instance a plugin name that don\'t exists';
```

Comment:
Will this expression fail if the error message fails next time ? Should we may be check the comparison to be exact ? Like `expect.().to.equal` ?

Reply:
With the current setup we only check if the substring is present in the complete error message (which is a little bit longer). We could also check for equality for the complete error message. But, IMO the substring mechanism should be fine, but also open to check for equality.